public/app/plugins/datasource/cloudwatch/components/shared/MetricStatEditor/MetricStatEditor.tsx

```diff
@@ -146,7 +146,24 @@ export const MetricStatEditor = ({
           <EditorField
             label="Match exact"
             optional={true}
-            tooltip="Only show metrics that exactly match all defined dimension names."
+            tooltip={
+              <>
+                {
+                  'Only show metrics that exactly match all defined dimension names. If this is enabled, you must specify all dimensions of the metric you’re querying so that the '
+                }
+                <a
+                  href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/search-expression-syntax.html"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  metric schema
+                </a>
+                {
+                  ' matches exactly. If this is disabled you can specify any number of dimensions on which you’d like to filter.'
```

Comment:
I guess reading this I'm still a bit unclear on this. Some questions: 
- If I specify one dimension as `instanceId: *` how does the match exact impact this query, if at all? What does it mean for a `*` to "match exactly"
- Is this like an `Or` vs `And` thing?  Like without match exact it's an `Or` so that any metrics with one of the dimensions matches will show and with match exact it's like an And where all dimensions have to match to show? 
- In that linked doc it's talking about partial vs exact matches example they give is that searching for "Custom" won't return "CustomCount1" but I guess searching for 'Custom' might? But I'm not sure how this comes into play with a query builder... like if we pre-populate dimension values with predefined things, selecting one of those options without match exact enabled means we might get more results yeah? Seems like an odd experience and I'm not clear why someone would want that? I guess in the case of a custom value we can't parse or something?

Reply:
So (while AWS doesn't define this anywhere and I think should) I'm pretty sure a `metric schema` is a metric name and all the dimension keys it has, and its exactly all the dimensions the metric has, no extras or missing dimensions. 

- The query has 2 parts, the `metric schema` and the `search term`, and the metric schema dictates the shape of the data. The second part, the `search term` checks what those values are set as. So `instanceId: *` adds instanceId to the `metric schema`, but doesn't affect the `search term`. The schema is what we are applying `match exact` to, when it's not enabled we don't send a schema to match.
- No, the way we construct the queries it's always an `And` thing. Without `match exact` we still care about the specified values all matching, but we don't care if there are other dimensions on the metric.
- We put all the values in `""` so the partial matches stuff doesn't come into play for us. I also don't know why that would be helpful

Is that clearer? Should I change the wording?

Reply:
Almost right! When match exact is disabled the results will include the match exact results and results that have the dimension keys and additional dimension keys. It won't return metrics that don't have the dimension keys (unless a different dimension has that key as its value).
I'm a little confused about the duplicate dimension names thing? Is this an issue we've run into? I guess they don't say that they have to be unique, but I don't see any examples of a metric with the same key for multiple dimensions in their docs.
I can work on adding more details to the docs.

Reply:
ah ok I think I get it now thanks!! And I think the docs make sense, although it's hard to say until I forget how this works again. Please ignore what I was saying about duplicates I think I was confusing myself while looking at something and coming up with false reasons that something wasn't returning data when match exact was on.